python/core/gps/qgsqtlocationconnection.sip

```diff
@@ -1,41 +1,70 @@
-%Feature MOBILITY_LOCATION
```

Comment:
A lot's been lost from this sip file - the MOBILITY_LOCATION stuff, and code which is blocked on android below